librz/analysis/dwarf_process.c

```diff
@@ -931,6 +989,8 @@ static const char *get_dwarf_reg_name(RZ_NONNULL char *arch, int reg_num, Variab
 		if (bits == 64) {
 			return map_dwarf_reg_to_ppc64_reg(reg_num, kind);
 		}
+	} else if (!strcmp(arch, "tricore")) {
+		return map_dwarf_reg_to_tricore_reg(reg_num, kind);
```

Comment:
I wonder whether we should add a new `dwarf_reg_to_reg` method to the `RzAnalysisPlugin` instead to make this extensible and generic.

Reply:
Not in this PR please. But we could think about this, yes.

Reply:
I agree, not in this PR. But it should be done if/before we are going to add another `if` here.